chaco/scatter_inspector_overlay.py

```diff
@@ -41,7 +42,7 @@ class ScatterInspectorOverlay(AbstractOverlay):
     # using a series of trait change handlers (defined at the end of the
     # class)
     #@on_trait_change('component.index.metadata_changed,component.value.metadata_changed')
-    def metadata_changed(self, object, name, old, new):
+    def metadata_updated(self, event):
```

Comment:
public API - just being safe

```suggestion
    def metadata_updated(self, event=None):
```